### DISCUSSION TITLE
Suggestions for Hacktoberfest contributions

### DISCUSSION THREAD

**Discussion by RommelTJ** [playframework/playframework#12891] [General]
Hi. I'm looking to contribute to Open Source projects for Hacktoberfest and since I use Play framework on a daily basis this seems like a great project to contribute to. 

I read the contribution guidelines, and I'm sorting through the issues labeled as "good first issues", but I'm wondering if the community here has a suggestion on where to start, or a particular issue they want to see addressed that is currently not being looked at. Some of the issues labeled as "good first issues" appear abandoned so want to make sure I work on something the community feels is valuable.

**Comment by mkurz** [comment]
Hi,

thanks for your interest! I am not sure how up to date the `good first issues` label is, so I scrolled through the first couple issue pages:

There was an issue that already had PR but got closed as it looks like the author lost interest, so I think this one might be a low hanging fruit:
- #10893 (abandoned PR: #11451)

If you are also interested in enhancing docs:
- #10572
- #10469

Also, not sure how complicated this one would be, but someone came up with a reproducer, so maybe that would help to figure out what's going on:
- #11600

**Reply by RommelTJ** [reply]
This is very helpful. I'll take a look. Thanks.

**Reply by RommelTJ** [reply]
Added some PRs for review: 
* https://github.com/playframework/playframework/pull/12892
* https://github.com/playframework/playframework/pull/12894
* https://github.com/playframework/playframework/pull/12895
* https://github.com/playframework/playframework/pull/12896

A `hacktoberfest-accepted` label would be appreciated. 😃

**Reply by mkurz** [reply]
@RommelTJ thanks a lot for your contributions. You probably have seen I finally merged almost all your pull requests, we only need to put the Guice one on hold because it looks like there is a bug in Guice itself we can not work around: https://github.com/playframework/playframework/pull/12894#issuecomment-2671286973
Thanks again!
